lib/rubocop/sketchup/cop/suggestions/command_title.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module SketchupSuggestions
+      # SketchUp command titles should be in title case, e.g. "Make Component"
+      # and "Follow Me".
+      #
+      # In English, capitalize the first letter of each word. Other languages
+      # may have different rules.
+      class CommandTitle < SketchUp::Cop
+
+        def_node_matcher :init_entity?, <<-PATTERN
+          (send (const (const nil? :UI) :Command ) :new ... )
+        PATTERN
+
+        MESSAGE = 'Use title case in command titles. '\
+                  'In English, capitalize the first letter of each word.'
+
+        def on_send(node)
+          return unless init_entity?(node)
+          return if node.arguments.first.str_content.nil?
+          return if title_case?(node.arguments.first.str_content)
+
+          add_offense(node, message: MESSAGE)
+        end
+
+        private
+
+        # REVIEW: Extract to where they can be re-used?
+        def title_case?(text)
+          text == title_case(text)
+        end
+
+        def title_case(text)
+          text.gsub(/^(.)/) { ::Regexp.last_match(1).upcase }
```

Comment:
Is this different from `string.capitalize`?
https://ruby-doc.org/core-2.7.1/String.html#method-i-capitalize

If it is, can you add a comment with examples of what it does? (Hard to tell from just glancing at the regexes)